workflow-ui/core-android/src/androidTest/java/com/squareup/workflow1/ui/container/DialogIntegrationTest.kt

```diff
@@ -79,12 +91,14 @@ internal class DialogIntegrationTest {
 
     scenario.onActivity { activity ->
       val root = WorkflowLayout(activity)
+      activity.setContentView(root)
```

Comment:
Oops, these tests were never actually attaching anything to the activity window. Now they do.